pkgs/top-level/python-packages.nix

```diff
@@ -6484,6 +6484,23 @@ in {
     };
   };
 
+  peewee = buildPythonPackage rec {
+    name = "peewee-2.8.8";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.